core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="admin-prod-mode-activities-tab">
```

Comment:
Why adding this extra div? (Also, for oppia specific classes please add a prefix `oppia`.)